README.md

```diff
@@ -7,7 +7,45 @@ This repository provides KubeVirt-specific Tekton tasks, which focus on:
 - Executing commands in VMs
 - Manipulating disk images with virt-customize [WIP]
 
-## Usage and Deployment
+## Deployment
+
+### On Kubernetes
+
+Run the following command to install KubeVirt Tekton tasks in an active namespace.
+
+The installation manifest includes required RBAC roles and service accounts.
+The following command needs to be run in every namespace the tasks will be used in, because these service accounts need to be created for each new namespace.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubevirt/kubevirt-tekton-tasks/main/manifests/kubernetes/namespace-rbac/kubevirt-tekton-tasks.yaml
```

Comment:
yes, I will do that once this PR merges - since I need the manifests introduced by this PR for the release. I will make additional PR which updates the readme after the release.